package.json

```diff
@@ -23,6 +23,8 @@
     "@popperjs/core": "^2.9.2",
     "@vue/web-component-wrapper": "^1.3.0",
     "core-js": "^3.6.5",
+    "fecha": "^4.2.1",
+    "imask": "^6.0.7",
```

Comment:
added `dependencies` must also be listed within `publish/package.json`

Reply:
Thanks, i will resolve it